public/src/utils/KeyManager.js

```diff
@@ -0,0 +1,34 @@
+export class KeyManager {
+  constructor(p5) {
+    this.keysPressed = [];
+    this.commands = [
```

Comment:
commands will live here - for now you can see each command consists of a `name`, `keys` array, `execute` command, and a single `param` that can optionally be passed to the `execute` function. we could potentially make it so it can accept more than one argument, but for now just keeping it simple.